torch/utils/cpp_extension.py

```diff
@@ -20,7 +20,7 @@
 from typing import List, Optional, Union
 
 from setuptools.command.build_ext import build_ext
-from pkg_resources import packaging, parse_version  # type: ignore[attr-defined]
+from pkg_resources import packaging  # type: ignore[attr-defined]
```

Comment:
let's see if it passes mypy
```suggestion
from pkg_resources import packaging
```